pkgs/desktops/gnome-3/extensions/dash-to-panel/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, glib, gettext }:
+
+stdenv.mkDerivation rec {
+  name = "gnome-shell-dash-to-panel-${version}";
+  version = "v11";
+
+  src = fetchFromGitHub {
+    owner = "jderose9";
+    repo = "dash-to-panel";
+    rev = version;
+    sha256 = "1bfcnrhw6w8yrz8sw520kwwshmplkg4awpvz07kg4d73m6zn4mw2";
+  };
+
+  nativeBuildInputs = [
```

Comment:
This should likely not be `nativeBuildInputs` but `buildInputs`.

This is related to cross compilation of nixpkgs.

`nativeBuildInputs` are for dependencies needed during the build process (think compilers, build tools etc) and `buildInputs` are for things that end up getting used on the target platform (think shared libraries and such needed at runtime).

Reply:
To be honest I just copied pretty much the whole of dash-to-dock, maybe that will also require an update then? Nonetheless, will fix it :)